src/main/java/com/st/utopia/cancellationservice/model/TicketIdentity.java

```diff
@@ -0,0 +1,95 @@
+package com.st.utopia.cancellationservice.model;
+
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+/**
+ * A class to be the primary key of a {@link Ticket} for JPA, which requires every
+ * Entity to have a single primary key.
+ *
+ * <p>Declared to implement Serializable because "Composite-id class must implement Serializable."
+ *
+ * @author Al-amine AHMED MOUSSA
+ */
+
+@Embeddable
+public class TicketIdentity implements Serializable {
+	/**
+	 * Serialization version. Increment on any change to class structure that is
+	 * pushed to production.
+	 */
+	private static final long serialVersionUID = 1L;
+	/**
+	 * the flight ID.
+	 */
+	@Column(name="flight")

```

Comment:
Don't need to give the column's name when it's the same as the field name (and it's not a `@JoinColumn`)